### DISCUSSION TITLE
Inconsistency between contribution graph and its links

### DISCUSSION THREAD

**Discussion by william-andre** [community/community#72724] [Repositories]
### Select Topic Area

Bug

### Body

When opening a contribution graph, for instance
https://github.com/odoo/odoo/graphs/contributors?from=2023-08-29&to=2023-10-27&type=c

And clicking on one of the links to the related commits, for instance
https://github.com/odoo/odoo/commits?author=xmo-odoo&since=2023-08-29&until=2023-10-27
But it is the same for any other contributor.

The number displayed are not the same: the graph page displays some commits (I see 6), but the commits page displays something else (I see 0)

I assume this is because the commit date might not be the same as the merge date?

**Comment by BhathiyaPrasad** [comment]
If you commit, when the pr merged you can get the contribution count so the commit count for x day = commit done that x day, and the pr merge commit calculate for the pr merge date
